### PR TITLE
Unify on one analysis_options file

### PR DESCRIPTION
**Pull request by kevmoo**
Apply fixes to shelf_router and shelf_router_generator

### CODE REVIEW

File: pkgs/shelf_router/lib/src/route.dart [hunk 1]
```diff
@@ -13,7 +13,7 @@
 // limitations under the License.
 
 import 'package:meta/meta.dart' show sealed;
-import 'package:shelf_router/src/router.dart';
+import 'router.dart';
```
Comment (devoncarew) [inline pkgs/shelf_router/lib/src/route.dart:16]: organize imports?

File: pkgs/shelf_router/lib/src/router.dart [hunk 2]
```diff
@@ -294,7 +295,7 @@ class _RouteNotFoundResponse extends Response {
   Response change({
     Map<String, /* String | List<String> */ Object?>? headers,
     Map<String, Object?>? context,
-    body,
+    Object? body,
```
Comment (devoncarew) [inline pkgs/shelf_router/lib/src/router.dart:298]: 👍

**Review by devoncarew — APPROVED** [review]

File: pkgs/shelf_router/lib/src/route.dart (hunk 1 above)
Comment (kevmoo) [reply pkgs/shelf_router/lib/src/route.dart:16]: prefer relative imports...

Comment (devoncarew) [reply pkgs/shelf_router/lib/src/route.dart:16]: For sure, but this changed the import from a package to a file import, and the 'organize import' command would insert a blank line between these two imports now :)

Comment (kevmoo) [reply pkgs/shelf_router/lib/src/route.dart:16]: ah! Hrm...yup

**Pull request merged**